Error: Cannot find module 'cssnano' when running CLI commands (alpha.5)

## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
Running `yarn develop` after upgrading the Getting Started update branch resulted in this error
```sh
npm run develop

> greenwood-getting-started@1.0.0 develop /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started
> greenwood develop

internal/modules/cjs/loader.js:797
    throw err;
    ^

Error: Cannot find module 'cssnano'
Require stack:
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-css.js
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/serve.js
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/commands/build.js
- /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-css.js:8:17)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/plugins/resource/plugin-standard-css.js',
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/serve.js',
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/commands/build.js',
    '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/index.js'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! greenwood-getting-started@1.0.0 develop: `greenwood develop`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the greenwood-getting-started@1.0.0 develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/owenbuckley/.npm/_logs/2021-02-20T16_42_06_532Z-debug.log
```

## Details
As part of the refactoring in #473 , with some of the plugin refactoring and shifting of _package.json_, it looks like `cssnano` is only coming in via **@greenwood/plugin-postcss** and it should just be default with **@greenwood/cli**
```sh
$ yarn why cssnano
yarn why v1.12.3
[1/4] 🤔  Why do we have the module "cssnano"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "cssnano@4.1.10"
info Reasons this module exists
   - "_project_#@greenwood#plugin-import-css#rollup-plugin-postcss" depends on it
   - Hoisted from "_project_#@greenwood#plugin-import-css#rollup-plugin-postcss#cssnano"
info Disk size without dependencies: "48KB"
info Disk size with unique dependencies: "1.36MB"
info Disk size with transitive dependencies: "25.89MB"
info Number of shared dependencies: 52
✨  Done in 1.01s.
```

Just need to explicitly add / hoist **cssnano** to the `@greenwood/cli` package.

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -30,6 +30,7 @@
     "acorn": "^8.0.1",
     "acorn-walk": "^8.0.0",
     "commander": "^2.20.0",
+    "cssnano": "^4.1.10",
     "es-module-shims": "^0.5.2",
     "front-matter": "^4.0.2",
     "htmlparser2": "^4.1.0",

